### PR TITLE
Fixes for notactive

### PR DESCRIPTION
**Pull request by lislis**

### CODE REVIEW

File: app/styles/app/layouts/missing-notice.sass [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
-  a
+  p a
+    color: $grey-dark
+    text-decoration: underline
+
+  a.button
     font-size: 14px
 
+
```
Comment (GrahamCampbell) [inline app/styles/app/layouts/missing-notice.sass]: ?

Comment (lislis) [inline app/styles/app/layouts/missing-notice.sass]: What's the question?

Comment (GrahamCampbell) [inline app/styles/app/layouts/missing-notice.sass]: Why adding the extra newline?

Comment (lislis) [inline app/styles/app/layouts/missing-notice.sass]: Ah, my bad. Didn't realize that. :+1:

**Pull request merged**